doc/source/doc-style/code/required_conf.py

```diff
@@ -0,0 +1,85 @@
+"""Sphinx documentation configuration file."""
+from datetime import datetime
+
+from ansys_sphinx_theme import pyansys_logo_black as logo
+
+# Project information
+project = "ansys-product-library"
+copyright = f"(c) {datetime.now().year} ANSYS, Inc. All rights reserved"
+author = "ANSYS, Inc."
+release = version = "0.1.dev0"
+
+# Select desired logo, theme, and declare the html title
+html_logo = logo
+html_theme = "ansys_sphinx_theme"
+html_short_title = html_title = "pyproduct-library"
+
+# specify the location of your github repo
+html_theme_options = {
+    "github_url": "https://github.com/pyansys/pyproduct-library",
+    "show_prev_next": False,
+    "show_breadcrumbs": True,
+    "additional_breadcrumbs": [
+        ("PyAnsys", "https://docs.pyansys.com/"),
+    ],
+}
+
+# Sphinx extensions
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.autosectionlabel",
+    "numpydoc",
+    "sphinx.ext.intersphinx",
+    "sphinx_copybutton",
+]
+
+# Intersphinx mapping
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/dev", None),
+    # kept here as an example
+    # "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
+    # "numpy": ("https://numpy.org/devdocs", None),
+    # "matplotlib": ("https://matplotlib.org/stable", None),
+    # "pandas": ("https://pandas.pydata.org/pandas-docs/stable", None),
+    # "pyvista": ("https://docs.pyvista.org/", None),
+    # "grpc": ("https://grpc.github.io/grpc/python/", None),
+}
+
+# numpydoc configuration
+numpydoc_show_class_members = False
+numpydoc_xref_param_type = True
+
+# Consider enabling numpydoc validation. See:
+# https://numpydoc.readthedocs.io/en/latest/validation.html#
+numpydoc_validate = True
+numpydoc_validation_checks = {
+    "GL06",  # Found unknown section
+    "GL07",  # Sections are in the wrong order.
+    "GL08",  # The object does not have a docstring
+    "GL09",  # Deprecation warning should precede extended summary
+    "GL10",  # reST directives {directives} must be followed by two colons
+    "SS01",  # No summary found
+    "SS02",  # Summary does not start with a capital letter
+    # "SS03", # Summary does not end with a period
+    "SS04",  # Summary contains heading whitespaces
+    # "SS05", # Summary must start with infinitive verb, not third person
+    "RT02",  # The first line of the Returns section should contain only the
+    # type, unless multiple values are being returned"
+}
```

Comment:
```suggestion
    "SS02",  # Summary does not start with a capital letter
    "SS03", # Summary does not end with a period
    "SS04",  # Summary contains heading whitespaces
    "SS05", # Summary must start with infinitive verb, not third person
    "RT02",  # The first line of the Returns section should contain only the
    # type, unless multiple values are being returned"
}
```

I would enforce all rules by default - not leave them commented out. However, if you feel that they are not being applied anywhere and we should just delete them, please go ahead =)

Reply:
Sure, no problem!